buildSrc/src/toolkits/gradle/GenerateSdk.kt

```diff
@@ -15,6 +15,8 @@ import software.amazon.awssdk.codegen.model.service.ServiceModel
 import software.amazon.awssdk.codegen.utils.ModelLoaderUtils
 import java.io.File
 
+/* ktlint-disable custom-ktlint-rules:log-not-lazy */
+
```

Comment:
nit: spacing, `Tasks.kt` has no blank line between the comment and the class, this does - I'm surprised ktlint didn't pick it up.

Reply:
Fixed that in the changelog refactor PR